.github/workflows/test_itensorgaussianmps.yml

```diff
@@ -14,7 +14,6 @@ jobs:
     strategy:
       matrix:
         version:
-          - '1.3'
```

Comment:
Instead of removing this we should change it to `1.6` so then we make sure to test Julia 1.6. Without it we will only test `1`, which defaults to the latest released version (so for example right now it default to Julia 1.7).

Reply:
Ok will make that change